grails-app/utils/au/org/emii/portal/HtmlCharacterEntityReferences.groovy

```diff
@@ -1,7 +1,10 @@
 /*
  * Copyright 2015 IMOS
  *
- * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ * The AODN/IMOS Portal is licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
By all means, I'm not a lawyer. It's just that if it was my code here in the file, this is what I would have expected.